src/llvm_interpreter.rs

```diff
@@ -229,7 +229,13 @@ fn add_c_declarations(module: &mut Module) -> CDeclarations {
 
     add_function(module,
                  "llvm.memset.p0i8.i32",
```

Comment:
Hm, this happens only when you use `fn_call_style = "Block"` which is nice but currently does very weird things in certain function calls. So we have to go with this kinda half hearted sometimes block, sometimes visual design.